browserscripts/browser/asyncAppConstants.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   requires: { privilege: true },
-  function: async function() {
+  pre: function() {},
```

Comment:
Can we have an example (or a new script?) with non-trivial `pre` and `post`?

Reply:
There is an example included in the docs/ directory. Would you like me to commit it to the repo?

Reply:
Yes, I want there to be an example (command-line-option-ed off) that's easy for everybody to run/copy.